cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrapSettings.scala

```diff
@@ -60,6 +60,17 @@ final class ClusterBootstrapSettings(config: Config) {
     val interval: FiniteDuration =
       discoveryConfig.getDuration("interval", TimeUnit.MILLISECONDS).millis
 
+    val exponentialBackoffFactor: Double =
+      discoveryConfig.getDouble("exponential-backoff-factor")
+
+    val exponentialBackoffMax: Duration =
```

Comment:
why not `FiniteDuration`?

Reply:
AFAIR one of the calculations returns a Duration so it bubbled up though realistically it should always be Finite ones, I'll change that